src/index.html

```diff
@@ -4,9 +4,456 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="/styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html" class="header__logo">
+        <img
+        src="images/logo.png"
+        alt="header logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">
+                apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">samsung</a>
+          </li>
```

Comment:
Let's specify some anchor link in all cases instead of using single # sign
```suggestion
          <li class="nav__item">
            <a href="#samsung" class="nav__link">samsung</a>
          </li>
```